Gemfile.lock

```diff
@@ -883,4 +883,4 @@ DEPENDENCIES
   webmock (~> 3.18, >= 3.18.1)
 
 BUNDLED WITH
-   2.5.6
+   2.3.27
```

Comment:
The bundler version in the docker file is bundler:2.5.6 
I don't know if this will cause any issues.

Reply:
I think Bundler grumbles but doesn't fail, but it would be nice to have this value consistent